.github/workflows/labels.yml

```
@@ -0,0 +1,128 @@
+          LABELS_JSON=$(cat .github/labels.json)
+          echo "labels<<EOF" >> $GITHUB_OUTPUT
+          echo "$LABELS_JSON" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+      
+      - name: Create labels
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const labelsJson = `${{ steps.labels.outputs.labels }}`;
+            const labels = JSON.parse(labelsJson);
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)